SU-SOE/stanford_soe_helper
# 7.x 2.x spotlight

READY FOR REVIEW

# Summary
- Spotlight Horizontal views

# Needed By (Date)


# Urgency
- Yes

# Steps to Test

- Pull this branch
- Clear cache
- Revert all _Stanford People Spotlight_ features
- Verify  all _Stanford People Spotlight_ features are in a default state ($ drush fd )
- Create 2 _Stanford People Spotlight_  nodes: 
- - one with "affiliation" student" 
- - one with "affiliation" faculty" 
- Place both all 6 (All, Student, and Faculty x  block and no block) _Stanford People Spotlight_  view blocks on  a page
- Verify that all display as expected 

# Affected Projects or Products

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-1939

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
